react/PhoneField.tsx

```diff
@@ -36,6 +36,7 @@ interface Props
   errorMessage?: string
   helpText?: React.ReactNode
   suffix?: React.ReactNode
+  size: 'small' | 'regular' | 'large'
```

Comment:
this should be an optional property
```suggestion
  size?: 'small' | 'regular' | 'large'
```

Reply:
nice catch! thx